docs/meshstack.identity-federation.md

```diff
@@ -365,3 +365,15 @@ let AzureLookupStrategy =
       < UserByMailLookupStrategy | UserByUsernameLookupStrategy >
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+## Group Cleanup
+
+For the platform listed below user access is given by assigning groups with the apropriate users inside to the platform tenant. The naming of these groups can be customized by setting the `User Group Name Pattern` in the appropriate meshPlatform Configuration. See the image below.
+
+![User Group Name pattern in the meshPlatform config](assets/platform-config-groupname-pattern.png)
+
+Additionally added group permissions are discouraged, but possible. During replication meshStack won't remove additionally added group permissions that external systems might have added.
```

Comment:
Recently I found that at least for Azure, we removing all customer-added permission for groups, and trying to restore the desired state from our role mappings. Not sure about the other platforms

Reply:
I adapted it to reflect our latest findings.